components/ai_chat/core/browser/engine/oai_api_client_unittest.cc

```diff
@@ -113,8 +113,11 @@ class OAIAPIUnitTest : public testing::Test {
 };
 
 TEST_F(OAIAPIUnitTest, PerformRequest) {
-  mojom::CustomModelOptionsPtr model_options = mojom::CustomModelOptions::New(
-      "test_api_key", GURL("https://test.com"), "test_model");
+  mojom::CustomModelOptionsPtr model_options = mojom::CustomModelOptions::New();
+
+  model_options->model_request_name = "test_model";
+  model_options->endpoint = GURL("https://test.com");
+  model_options->api_key = "test_api_key";
```

Comment:
The pattern that's been mostly enforced is to specify mojom object construction params. Although not as pretty, the intention is to get a compiler error when adding non-optional properties to the mojom type.

Reply:
Does this pattern (construction params) apply equally to tests as it does to project code? Either way, I'll make the necessary changes :)